packages/common/src/configuration/service-configuration.ts

```diff
@@ -266,6 +267,11 @@ export class ServiceConfiguration {
                     default: 'https://www.washington.edu/accesscomputing/AU/before.html',
                     doc: 'Url to scan for availability testing',
                 },
+                consolidatedReportId: {
+                    format: 'String',
+                    default: 'somerandomid',
```

Comment:
we could have the report id with more meaningful name like e2e-consolidated-report-id or similar

Reply:
changed